Backend/KFC_WebAPI/Controllers/LogoutController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Data.Entity.Infrastructure;
+using System.Net;
+using System.Web.Http;
+using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using ManagerLayer;
+using ServiceLayer.Services;
+
+
+
+namespace KFC_WebAPI.Controllers
+{
+
+
+    public class LogoutRequest
+    {
+        [Required]
+        public string token { get; set; }
+    }
+
+    public class LogoutController : ApiController
```

Comment:
No need for a whole controller just for logout. This should be in the UserController